src/components/MdTable/MdTable.vue

```diff
@@ -222,11 +222,28 @@
           this.MdTable.hasValue = this.hasValue
         }
       },
-      'MdTable.selectedItems' (val) {
-        this.select(val)
+      'MdTable.selectedItems' (val, old) {
```

Comment:
I think this method is way complex. Maybe we can break it in some small functions to better describe it.

Reply:
Like: `val.length !== old.length) ? true : !val.every((item, index) => item == old[index]`